src/gsi/get_gefs_ensperts_dualres.f90

```diff
@@ -392,8 +316,17 @@ subroutine get_gefs_ensperts_dualres
            en_perts(n,m)%valuesr4(i)=en_perts(n,m)%valuesr4(i)*sig_norm
         end do
      end do
+  end do  ntlevs_ens_loop !end do over bins
+
+  do n=n_ens,1,-1
+     call gsi_bundledestroy(en_read(n),istatus)
+     if ( istatus /= 0 ) &
+        call die('get_gefs_ensperts_dualres',': trouble destroying en_read bundle, istatus = ', istatus)
   end do
+  deallocate(en_read)
+  call gsi_enscoupler_destroy_sub2grid_info(grd_tmp)
 
+! mm1=mype+1
```

Comment:
Comments here may answer question asked above.  Seems we retain this code as a placeholder for possible future development.   If not a placeholder for future development, we could remove the commented out code.

Reply:
Same reply as above.  Xu can decide what to do later.